addons/hr/static/src/components/avatar_card/avatar_card_popover_patch.js

```diff
@@ -3,34 +3,52 @@
 import { patch } from "@web/core/utils/patch";
 import { AvatarCardPopover } from "@mail/discuss/web/avatar_card/avatar_card_popover";
 import { useService } from "@web/core/utils/hooks";
+import { _t } from "@web/core/l10n/translation";
 
 export const patchAvatarCardPopover = {
     setup() {
         super.setup();
-        this.userInfoTemplate = "hr.avatarCardUserInfos",
+        this.userInfoTemplate = "hr.avatarCardUserInfos";
         this.actionService = useService("action");
     },
-    get fieldNames(){
-        let fields = super.fieldNames;
+    get fieldNames() {
+        const fields = super.fieldNames;
         return fields.concat([
             "work_phone",
-            "work_email", 
-            "job_title", 
-            "department_id", 
+            "work_email",
+            "name_work_location_display",
+            "type_work_location",
+            "job_title",
+            "department_id",
             "employee_ids",
         ]);
     },
-    get email(){
+    get email() {
         return this.user.work_email || this.user.email;
     },
-    get phone(){
+    get phone() {
         return this.user.work_phone || this.user.phone;
     },
-    async onClickViewEmployee(){
+    get name_work_location_display() {
+        return this.user?.name_work_location_display === _t("Unspecified")
```

Comment:
Not great to compare to translated value, can't we send false/undefined when there is no value instead?